src/main/java/org/apache/rocketmq/dashboard/config/PoolConfig.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.rocketmq.dashboard.config;
+
+import org.apache.commons.pool2.impl.GenericObjectPool;
+import org.apache.commons.pool2.impl.GenericObjectPoolConfig;
+import org.apache.rocketmq.dashboard.admin.MQAdminFactory;
+import org.apache.rocketmq.dashboard.admin.MQAdminPooledObjectFactory;
+import org.apache.rocketmq.tools.admin.MQAdminExt;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class PoolConfig {
+
+    @Autowired
+    private RMQConfigure rmqConfigure;
+
+    @Bean
+    public GenericObjectPool<MQAdminExt> mqAdminExtPool() {
+        GenericObjectPoolConfig genericObjectPoolConfig = new GenericObjectPoolConfig();
+        genericObjectPoolConfig.setTestWhileIdle(true);
+        genericObjectPoolConfig.setMaxTotal(3);
```

Comment:
Is 3 enough? Could you make it configurable?

Reply:
I tested that most of the time there were no more than 3 objects in the object pool, so I used the default value of 8 instead of making it a configurable item